public/app/features/explore/slate-plugins/clear.test.tsx

```diff
@@ -0,0 +1,42 @@
+import Plain from 'slate-plain-serializer';
+import React from 'react';
+import { Editor } from '@grafana/slate-react';
```

Comment:
What's the plan for maintaining this vs. eventually going back to non-forked version?

Reply:
I think we can switch back to the non-forked version as soon as my fixes get merged, but unfortunately things move fairly slowly upstream